src/Physics/Resonance/XSection/BostedChristyEMPXSec.cxx

```diff
@@ -236,11 +237,14 @@ void BostedChristyEMPXSec::FermiSmearingA(double Q2, double W, double pF, double
   double dW2dpF = 2.*qv;
   double dW2dEs = 2.*(nu + MN); 
   // switched to using 99 bins!
-
-  for (int i=0; i<100; i++)
+  F1p = 0;
+  F1d = 0;
+  sigmaT = 0;
+  sigmaL = 0;
+  for (int i=0; i<99; i++)
```

Comment:
These seem to make sense, and definitely fix the previous version's overrun of the array.